_posts/2022-11-07-why-python-package-health-metrics-matter.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: single
+title: "Why Thinking About Metrics Should be the First Thing That Any New Organization, Program or Project Does"
+excerpt: "Collecting baseline data that aligns with the goals and outcomes of your project, program or organization is critical to do at the beginning. Here I briefly explain why Python package health metrics are so important to the long-term success of pyOpenSci."
+author: "Leah Wasser"
+permalink: /blog/why-python-package-health-metrics-matter.html
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - blog-post
+  - highlight
+  - python-packaging
+  - peer-review
+toc: true
+comments: true
+---
+
+[In the last blog post,](/blog/why-python-open-source-software-matters-for-scientists.html) 
+I discussed why the health of (Python) open source 
+packages should matter to you as a scientist (and as a person who values and 
+uses free and open source tools in your workflow). In this post, I'll talk
+more about why collecting metrics are critical to both program development success
+and to the success of open source tools. I'll wrap this three-part blog series 
+up [with a discussion on what types of package metrics pyOpenSci should be collecting 
```

Comment:
Is this link broken because it spread across multiple lines?
Maybe change to 
```
I'll wrap this three-part blog series up with a [last post](link) discussing what types of package metrics pyOpenSci should be collecting ...
```

Reply:
it is. i don't like short links where the text within the a href doesn't describe what the actual link is for.